assignments/models.py

```diff
@@ -365,6 +369,44 @@ def upload_graded_submissions_to_canvas(self,
                 if uploaded:
                     print(f"Uploaded file comment to canvas for {canvas_submission.user['name']}")
 
+
+class Version(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=255)
+    assignment = models.ForeignKey(Assignment, on_delete=models.CASCADE)
+    versions = models.CharField(max_length=255, null=True, blank=True, default=None)
```

Comment:
Do we need a field versions in Version? I haven't read all the files yet, but I don't get its semantic meaning.